File: beginner/java/1190.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        double[][] m = new double[12][12];
        int i, j;
        char operacao;

        operacao = scanner.next().charAt(0);

        for(i = 0; i < 12; i++){
            for(j = 0; j < 12; j++){
                m[i][j] = scanner.nextDouble();
            }
        }
        
        System.out.printf("%.1f%n", soma_or_media(m, operacao));



        scanner.close();
    }

    public static double soma_or_media(double[][] mat, char op){
        int i, j, cont = 0;
        double soma = 0.0;

        for(i = 0; i < 12; i++)
			for(j = 0; j < 12; j++)
				if (i < j && i > 12 - j - 1){
					soma += mat[i][j];
                    cont++;
                }
    
        
        if(op == 'S'){
            return soma;
        } else{
            return soma/cont;
        }
    }
}

File: beginner/c/1190.c
#include <stdio.h>

int main() {

  double m[12][12], soma = 0.0;
  int i, j, cont = 0, p = 10, r = 7;
  char operacao;

  scanf(" %c", &operacao);

  for (i = 0; i < 12; i++) {
    for (j = 0; j < 12; j++) {
      scanf("%lf", &m[i][j]);
    }
  }

  for(i = 0; i < 12; i++)
			for(j = 0; j < 12; j++)
				if (i < j && i > 12 - j - 1){
					soma += m[i][j];
          cont++;
        }

  if (operacao == 'S') {
    printf("%.1lf\n", soma);
  } else {
    printf("%.1lf\n", soma / cont);
  }

  return 0;
}
